system.go

```diff
@@ -22,6 +22,19 @@ import (
 	"github.com/winlabs/gowin32/wrappers"
 )
 
+func InitiateShutdown(machineName string, message string, gracePriod int, shutdownFlags int, reason int) error {
+	if err := wrappers.InitiateShutdown(

```

Comment:
Also, let's add enumerations for the `shutdownFlags` and `reason` fields.

Reply:
what about recent changes?